### PR TITLE
some cleanup and addition of a glossary

### PR DESCRIPTION
**Pull request by jonwinton**
- Adds an initial glossary
- Updates the homepage

### CODE REVIEW

File: website/siteConfig.js [hunk 1]
```diff
@@ -8,36 +8,31 @@
...
   headerLinks: [
-    {doc: 'components', label: 'Components'},
-    {doc: 'pages', label: 'Pages'},
-    {doc: 'layouts', label: 'Layouts'},
-    {doc: 'uris', label: 'Uris'},
-    {doc: 'lists', label: 'Lists'},
-    {doc: 'users', label: 'Users'},
-    {href: repoUrl, label: 'GitHub'}
+    {doc: 'getting-started', label: 'Getting Started'},
+    {href: repoUrl, label: 'G'}
```
Comment (oscarpolanco) [inline website/siteConfig.js]: Can we put `Github` as a label here 😅

File: docs/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+id: getting-started
+title: Getting Started
+sidebar_label: Getting Started
+---
+
+> The docs are being worked on and more content will be added soon!
+
+Below are some resources for getting started with Clay. To get started with a simple Clay instance you can [check out the starter repo](https://github.com/clay/clay-starter). If you see a phrase you don't recognize then check the [Glossary](/docs/glossary).
+
+If there'a anything not covered in these docs you explicitly want to see [then file an issue](https://github.com/clay/clay/issues) and we'll get on it.
```
Comment (oscarpolanco) [inline docs/getting-started.md]: I think this should be `If there's` instead of `If there'a`. What do you think?

File: docs/users.md [hunk 3]
```diff
@@ -1,12 +1,12 @@
 ---
 id: users
 title: Users
-sidebar_label: users
+sidebar_label: Users
```
Comment (oscarpolanco) [inline docs/users.md:4]: 👍

File: docs/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+---
+id: getting-started
+title: Getting Started
+sidebar_label: Getting Started
+---
+
+> The docs are being worked on and more content will be added soon!
+
+Below are some resources for getting started with Clay. To get started with a simple Clay instance you can [check out the starter repo](https://github.com/clay/clay-starter). If you see a phrase you don't recognize then check the [Glossary](/docs/glossary).
+
```
Comment (oscarpolanco) [inline docs/getting-started.md:10]: We got `404` in this links 😅 
You can use the name of the docs on the links
`[Glossary](glossary.md)`

File: docs/getting-started.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+If there'a anything not covered in these docs you explicitly want to see [then file an issue](https://github.com/clay/clay/issues) and we'll get on it.
+
+## Data Structures
+
+- [Components](/docs/components): the backbone of Clay. Almost everything is a component!
+- [Pages](/docs/pages): contains references to components
+- [Layouts](/docs/layouts): similar to components, but instances are meant to be shared across pages
+- [Uris](/docs/uris): a mapper from public urls to Clay uris
+- [Lists](/docs/lists): a store for simple JSON objects that you can store data easily for components to reference
+- [Users](/docs/users): the users of Clay who make the content
```
Comment (oscarpolanco) [inline docs/getting-started.md]: Need to update these links too

File: website/pages/en/index.js [hunk 6]
```diff
@@ -99,25 +95,85 @@ class Index extends React.Component {
...
+              contents={[
+                {
+                  content: `Clay has _zero_ assumptions about your content, everything is simply a component. YOU give your components meaning that match your needs rather than forcing your content into a content type`,
+                  image: `${siteConfig.baseUrl}img/markdown.png`,
+                  imageAlign: 'top',
+                  imageAlt: 'TBD',
+                  title: 'Apply Your Own Content Model',
+                },
+                {
+                  content: `Clay is designed to support multiple sites ina single instance meaning your components can be shared and re-used across sites within the same codebase`,
```
Comment (oscarpolanco) [inline website/pages/en/index.js]: Hey I think you can update `ina` to `in a` here 😅

File: website/pages/en/index.js [hunk 7]
```diff
@@ -99,25 +95,85 @@ class Index extends React.Component {
...
+                }
+              ]}
+              layout="twoColumn"
+            />
+          </Container>
+          <Container padding={['bottom', 'top']}>
+            <GridBlock
+              contents={[
+                {
+                  content: `Leave long forms in the past and edit your content on the same page it live. Clay's editing experience allows users to preview their changes as they happen and to know how the page will look without switching to another view.`,
```
Comment (oscarpolanco) [inline website/pages/en/index.js]: You can update the `it live` to `it lives`. What do you think?

**Review by oscarpolanco — COMMENTED** [review]
Very nice update 😄 
Just some minors comments

File: website/siteConfig.js (hunk 1 above)
Comment (jonwinton) [reply website/siteConfig.js]: Whoops, fixed.

**Review by oscarpolanco — APPROVED** [review]
Nice job!! LGTM

**Pull request merged**